docs/rest_api.rst

```diff
@@ -378,6 +378,7 @@ Channel Management
    :statuscode 409:
     - Provided channel does not exist or
     - ``state`` and ``total_deposit`` have been attempted to update in the same request.
+   :statuscode 417: Attempt to deposit more tokens than the testing limit, or attempt to deposit lower amount than on-chain balance of the channel.
```

Comment:
Can you also use 409. We use 409 everywhere as the "invalid/illegal" input error code.